src/arduino/arduinoSettings.ts

```diff
@@ -214,6 +214,8 @@ export class ArduinoSettings implements IArduinoSettings {
             this._packagePath = path.join(this._arduinoPath, "portable");
         } else if (util.fileExistsSync(path.join(this._arduinoPath, "AppxManifest.xml"))) {
             this._packagePath = path.join(folder, "ArduinoData");
+        } else if (process.env.hasOwnProperty('ARDUINO_DIRECTORIES_DATA')) {

```

Comment:
look like the lint does not approve single quotes, please fix them to double quotes.
```suggestion
        } else if (process.env.hasOwnProperty("ARDUINO_DIRECTORIES_DATA")) {

```